docs/conf.py

```diff
@@ -16,9 +16,7 @@
 from pathlib import Path
 import shutil
 import sys
-sys.path.insert(0, os.path.abspath('..'))
-
-import openfe_gromacs
```

Comment:
It's not necessary but we will eventually want it under version (below), do we know why it was failing?